### PR TITLE
[SECURITY-1686] Disable warning for 2.5.1+

### PR DESCRIPTION
**Pull request by daniel-beck**
CC @Igor-Filin

We'll need to confirm the issue is fully resolved first, and if it is, we'll merge this to disable the active security warning.

### CODE REVIEW

**Comment by daniel-beck** [comment]
Fixed in 2.5.2 according to maintainers.

**Comment by daniel-beck** [comment]
Can be merged once the PR build passes.

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**
